var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -27,13 +27,14 @@ class Magma(CMakePackage):
             description='Enable Fortran bindings support')
     variant('shared', default=True,
             description='Enable shared library')
+    variant('cuda', default=True)  # Cuda is a requirement
+    variant('cuda_arch', default='60', multi=True)
```

Comment:
Why set a default? I think we should leave it up to the user to decide

Reply:
I thought having cuda_arch=none is not option, since if that's past as a gpu_target, the build will/should fail. May be I should use a conflict statement instead to force user to choose something.